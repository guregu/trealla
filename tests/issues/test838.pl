:- initialization(main).
:- use_module(library(clpz)).

impossibilitas :- #X #> #Y, #Y #> #X.

main :-
	call_residue_vars(impossibilitas,Vs),
	write(Vs), nl.
