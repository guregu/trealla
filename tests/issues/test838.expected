[_9,_10]
